### PR TITLE
New tokens: week of September 24, part 2

### PR DESCRIPTION
**Pull request by therealMikkoK**
New one, like this

This PR:
* Inherits from #1718.
* Related to #1272 (also #1406, #1428).
* Related to #1025, #1129, #1231, #1253, #1695.

### CODE REVIEW

**Comment by freeatnet** [comment]
@therealMikkoK Looks good.

This PR:
* Inherits from #1718.
* Closes #1272 (also closes #1406, #1428).
* Closes #1025, #1129, #1231, #1253, #1695.

File: tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+addr: '0xcfc2437916a6df165235272dbfb116687bb1a00b'
+decimals: 18
+description: >-
+  A DECENTRALIZED CASHBACK PLATFORM ON THE ETHEREUM BLOCKCHAIN
```
Comment (freeatnet) [inline tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml]: Could this perhaps use a bit more description? What does the token do?

File: tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+links:
+- Facebook: https://www.facebook.com/designmast
```
Comment (freeatnet) [inline tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml]: I don't think this is an official link 😂

File: tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- Facebook: https://www.facebook.com/dsplus.fork
+- Github: https://github.com/dsplus/PlusCoin
+- Telegram: https://t.me/Dsplusforkeng
+- Telegram: https://t.me/PlusCoinENG
+- Telegram: https://t.me/joinchat/DdrYdkMYRX9wWiwPgjjxqg
+- Twitter: https://twitter.com/dsplusf
+- Twitter: https://twitter.com/dsplusf?lang=ru
```
Comment (freeatnet) [inline tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml]: Duplicate of the above, probably.

File: tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- Twitter: https://twitter.com/dspluslusirk
```
Comment (freeatnet) [inline tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml]: Also a personal link, it seems.

File: tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml (hunk 3 above)
Comment (therealMikkoK) [reply tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml]: Russian version, and English speaking. Should we leave both?

File: tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml (hunk 4 above)
Comment (therealMikkoK) [reply tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml]: remove link

**Comment by therealMikkoK** [comment]
@freeatnet should I review this, or what was the plan? Did you have things to do before we update?

**Comment by therealMikkoK** [comment]
Token file merge

**Pull request merged**